src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -66,7 +66,11 @@ def is_slack_enabled():
             raise
 
     def flush(self) -> None:
+
+        region = settings.SENTRY_REGION_OR_CUSTOMER
+
         json_data = {
+            "region": region,
```

Comment:
Ah I'll add the deployment process to the slack as well but: 
I made it so that if region is not given eng pipes prints messages as is, that way deployment order is not important. 

But Ideally, we would deploy here first, eng-pipes would ignore the extra key, I already tested this. 
Then we would deploy eng-pipes. 
Then we would add the webhook url to getsentry for ST. 
Finally we would turn on the option for the slack_webhook in ST.